homeassistant/components/climate/strings.json

```diff
@@ -25,5 +25,21 @@
       "dry": "Dry",
       "fan_only": "Fan only"
     }
+  },
+  "state_attributes": {
+    "_": {
+      "preset_mode": {
+        "values": {
```

Comment:
The schema allows providing name of the attribute, and the possible values.
Maybe only translations of the possible values are needed?

Reply:
The name itself is "generated" right now in the frontend, I think it is way better if we can provide a translation for them.

This is what we do now 😬 

https://github.com/home-assistant/frontend/blob/5c16447eedf41c1db5e4b48fc1dc01dbfc184792/src/data/entity_attributes.ts#L37-L45